src/utils/ironfish.ts

```diff
@@ -0,0 +1,20 @@
+import { IronfishSdk, RpcClient } from "@ironfish/sdk";
```

Comment:
Consider this an MVP to talk to the client. I'm sure this file will grow over time, but I figured starting small makes sense for now.

Reply:
Instead of making this use a local node, we should define it to run via TCP, that way we can easily hot swap nodes.
https://github.com/iron-fish/ironfish/blob/02939ae309a98205781908fd31e218aee4e62fea/ironfish/src/rpc/clients/tcpClient.ts#L7-L16

Reply:
The running node will need to be run with:
`--rpc.tcp` flag